src/store/index.ts

```diff
@@ -22,7 +25,14 @@ export const useStore = () => {
 export const store = createStore({
     state: {
         isAuthenticated: false,
-        recentSearches: get<SearchData<Identity>[]>(StoreKey.RecentSearches) ?? []
+        recentSearches: get<SearchData<Identity>[]>(StoreKey.RecentSearches) ?? [],
+        pagination: {
```

Comment:
@sosan-neikbeen Because that pagination is related to the identity search, we should rename it to e.g. ```identitySearchPagination```